code/game/objects/effects/spawners/random/pool/space_loot.dm

```diff
@@ -0,0 +1,282 @@
+/datum/spawn_pool/spaceloot
+	id = "space_loot_spawn_pool"
+	available_points = 1800
+
+/obj/effect/spawner/random/pool/spaceloot
+	icon = 'icons/effects/random_spawners.dmi'
+	icon_state = "giftbox"
+	spawn_pool_id = "space_loot_spawn_pool"
+	record_spawn = TRUE
+
+/obj/effect/spawner/random/pool/spaceloot/record_item(type_path_to_make)
+	if(ispath(type_path_to_make, /obj/effect))
+		return
+
+	SSblackbox.record_feedback("tally", "space_loot_spawns", 1, "[type_path_to_make]")
+
+/obj/effect/spawner/random/pool/spaceloot/dvorak_core_table
+	point_value = 100
+	guaranteed = TRUE
+	loot = list(
+		/obj/item/rcd/combat,
+		/obj/item/gun/medbeam,
+		/obj/item/gun/energy/wormhole_projector,
+		/obj/item/storage/box/syndie_kit/oops_all_extraction_flares,
+	)
+
+/obj/effect/spawner/random/pool/spaceloot/dvorak_emp_loot
+	point_value = 50
+	guaranteed = TRUE
+	loot = list(
+		/obj/item/grenade/empgrenade = 8,
+		/obj/item/gun/energy/ionrifle/carbine = 1,
+		/obj/item/gun/energy/ionrifle = 1,
+	)
+
+/obj/effect/spawner/random/pool/spaceloot/dvorak_displaycase
+	point_value = 100
+	guaranteed = TRUE
+	loot = list(/obj/structure/displaycase/dvoraks_treat)
+
+/obj/effect/spawner/random/pool/spaceloot/syndicate/common
+	name = "syndicate depot loot, common"
+	icon_state = "loot"
+	point_value = 10
+	loot = list(
+		// Loot schema: costumes, toys, useless gimmick items
+		/obj/item/clothing/mask/gas/syndicate,
+		/obj/item/clothing/shoes/magboots/syndie,
+		/obj/item/clothing/under/syndicate,
+		/obj/item/coin/antagtoken/syndicate,
+		/obj/item/deck/cards/syndicate,
+		/obj/item/soap/syndie,
+		/obj/item/storage/box/syndie_kit/space,
+		/obj/item/storage/fancy/cigarettes/cigpack_syndicate,
+		/obj/item/storage/secure/briefcase/syndie,
+		/obj/item/storage/toolbox/syndicate,
+		/obj/item/suppressor,
+		/obj/item/toy/syndicateballoon,
+	)
+
+/obj/effect/spawner/random/pool/spaceloot/syndicate/common/depot
+	spawn_inside = /obj/structure/closet/secure_closet/depot
+	spawn_loot_chance = 40
+
+/obj/effect/spawner/random/pool/spaceloot/syndicate/stetchkin
+	name = "syndicate depot loot, 20pct stetchkin"
```

Comment:
"20pct"? not too important since this is a spawner, not player-facing

Reply:
ah yeah, as in 20 percent chance of spawning. but we don't use that abbreviation elsewhere so i changed it to "%" to be more clear.